openid-4-verifiable-presentations-1_0.md

```diff
@@ -462,9 +462,7 @@ Other specifications can define further values for the `client_id_scheme` parame
 
 # Response {#response}
 
-A VP Token is only returned if the corresponding Authorization Request contained a `presentation_definition` parameter, a `presentation_definition_uri` parameter, or a `scope` parameter representing a Presentation Definition (#vp_token_request).
```

Comment:
I'm not really convinced that this sentence is unnecessary, and in fact I'd more on the side of thinking it's helpful. If others are convinced it is superfluous then fine, I'll not block this, but I thought it was worth stating why I've not added a positive review.

Reply:
yeah, honestly, I am not super convinced it is redundant either..

Reply:
We can close this PR if you think it is not redundant which I still think it is.